functions/transformations/filter_test.go

```diff
@@ -346,7 +346,7 @@ func TestFilter_NewQuery(t *testing.T) {
 			Name: "from with database filter with no parens including regex and field",
 			Raw: `from(bucket:"mybucket")
 						|> filter(fn: (r) =>
-							r["t1"]==/val1/
+							r["t1"]==/^val1/
```

Comment:
Did megacheck complain about this? That seems odd since this is an opaque string literal according to Go.